ssl/tls13_client.cc

```diff
@@ -889,6 +890,37 @@ static enum ssl_hs_wait_t do_send_client_certificate_verify(SSL_HANDSHAKE *hs) {
   return ssl_hs_error;
 }
 
+static enum ssl_hs_wait_t do_process_pha(SSL_HANDSHAKE *hs) {
+  SSL *const ssl = hs->ssl;
+
+  if(ssl->s3->pha_ext == SSL_PHA_EXT_SENT && ssl->s3->pha_enabled == 1) {
```

Comment:
`pha_enabled` is already a boolean, we shouldn't need to do this sort of comparison.